### PR TITLE
add th_TH translation

### LINKED ISSUE #91: Support Thai language
What problem does this feature solve?

I need react-component pagination support thai language.

What does the proposed API look like?

I will translate and pull request to complete this issue.

### PR DESCRIPTION
**Pull request by koobitor**
resolved #91 
@afc163 Could you merge th_TH to the master branch.

### CODE REVIEW

File: tests/Pagination.spec.js [hunk 1]
```diff
@@ -23,9 +23,10 @@ describe('Uncontrolled Pagination', () => {
   function shouldHighlightRight() {
     const pagers = TestUtils.scryRenderedDOMComponentsWithTag(pagination, 'li');
     const current2 = pagination.state.current;
+    const totalText = 1;
```
Comment (benjycui) [inline tests/Pagination.spec.js]: Why do you edit the test case?

Comment (koobitor) [inline tests/Pagination.spec.js]: I think some thing wrong on the test case.
Why master branch don't pass the 'npm run chrome-test'.
I try to complete my pull request and found the test case is not pass in the master.
Do you want me to fix the master code make it pass test case don't change the test case right?
Can you suggest me @benjycui am a beginner on this.

Comment (koobitor) [inline tests/Pagination.spec.js]: @benjycui i can rebase it remove the last commit that relate with the test case.
Then this pr will not pass The Travis CI.

**Comment by koobitor** [comment]
@benjycui Already fix the commit and change the total text element form li to span.

File: src/Pagination.jsx [hunk 2]
```diff
@@ -402,15 +402,15 @@ export default class Pagination extends React.Component {
 
     if (props.showTotal) {
       totalText = (
-        <li className={`${prefixCls}-total-text`}>
+        <span className={`${prefixCls}-total-text`}>
```
Comment (benjycui) [inline src/Pagination.jsx]: Nope...

**Comment by benjycui** [comment]
I had fix the CI, just rebase.

**Comment by koobitor** [comment]
@benjycui done.

**Comment by afc163** [comment]
> just rebase.

**Comment by koobitor** [comment]
@benjycui @afc163 Thank you so much :)

**Comment by benjycui** [comment]
+ rc-pagination@1.9.9

**Pull request merged**